ktxspec.adoc

```diff
@@ -194,20 +192,22 @@ After inflation from supercompression or when `supercompressionScheme ==
 .levelImages Structure
 [source, c]
 ----
+align( lcm(texel block size, 4) ) mipPadding <1>
```

Comment:
This line implies that the alignment is 4 for supercompressed textures, but I think the desired behavior (spelled out below) is that there's no alignment. Should this say "sgdByteLength > 0 ? align(...)"?

Reply:
Line 195 is in the levelImages structure section about which the immediately preceding text says 

>After inflation from supercompression or when `supercompressionScheme ==
0`, `levelImages` looks like this: